warehouse/models/mart/gtfs_schedule_latest/_gtfs_schedule_latest.yml

```diff
@@ -322,6 +322,46 @@ models:
         description: '{{ doc("gtfs_frequencies__exact_times") }}'
 
 
+  - name: dim_gtfs_datasets_latest
+    description: |
+      This table is a cut of the cleaned metadata representing GTFS datasets currently
+      active within the Cal-ITP ecosystem. Each record represents a GTFS dataset (feed)
+      that is either a type of GTFS Schedule, Trip Updates, Vehicle Locations or Alerts,
+      and provides base64-encoded URLs used to access that feed.
+    meta:
+      ckan.authority: https://calitp.org/
+    columns:
+      - name: name
+        description: |
+          An arbitrary human-readable title for the GTFS feed in question, usually
+          referencing the corresponding agency transit agency and feed type.
+      - name: type
+        description: |
+          GTFS data type ("schedule", "trip_updates",
+          "vehicle_positions", or "service_alerts")
+      - name: regional_feed_type
+        description: |
+          Describes whether this feed is a combined regional feed or has a relation
+          to a combined regional feed in some manner.
+          For example for MTC 511, the combined regional feed has type "Combined
+          Regional Feed", and the MTC-published subfeeds have type "Regional Subfeed".
+          If you are performing an analysis where using a regional combined feed is
+          inappropriate (even though that is the customer-facing data), this field can
+          help you assess other alternative feeds for the same services and organizations.
+          Not specified (null) for feeds with no relationship to regional feeds.
+      - name: uri
+        description: |
+          Plaintext URI of the GTFS feed referenced by the row, including templated
+          URL parameters for access, if relevant.
```

Comment:
was there discussion of `uri` vs `pipeline_url`? We may want to provide both, as `uri` is the legacy column

Reply:
The request we're fulfilling is better served by directly decoding `base64url_url` anyways, so I've gone that route instead.